source/_remoteClient/localMachine.py

```diff
@@ -251,9 +282,51 @@ def sendSAS(self) -> None:
 
 		:note: SendSAS requires UI Access. If this fails, a warning is displayed.
 		"""
-		if hasUiAccess():
+		if self._canSendSAS():
 			ctypes.windll.sas.SendSAS(0)
 		else:
 			# Translators: Message displayed when a remote computer tries to send ctrl+alt+del but UI Access is disabled.
 			ui.message(_("Unable to trigger Alt Control Delete from remote"))
```

Comment:
```suggestion
			ui.message(_("Unable to trigger Control Alt Delete from remote"))
```

Reply:
The item in the tool menu is instead "Send ctrl+alt+del". Maybe we should look at all the places where this keystroke is used (user guide, GUI, messages) and make something consistent:
* usage of "+" between keys
* usage of "del" or "delete"?